draft-ietf-tls-external-psk-importer.md

```diff
@@ -109,23 +112,29 @@ identities if imported incorrectly. Endpoints may incrementally deploy PSK Impor
 by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
 are distinct since their identities are different on the wire. See {{rollout}} for more details.
 
-Endpoints which import external keys MUST NOT use either the external keys or the derived
-keys for any other purpose. Moreover, each external PSK MUST be associated with at most
-one hash function, as per the rules in Section 4.2.11 from {{!RFC8446}}.
-See {{security-considerations}} for more discussion.
+Endpoints which import external keys MUST NOT use the keys that are input to the
+import process for any purpose other than the importer, and MUST NOT use the derived
+keys for any purpose other than TLS PSKs. Moreover, each external PSK fed to the
+importer process MUST be associated with at most one hash function, as per the
+rules in Section 4.2.11 from {{!RFC8446}}. See {{security-considerations}} for
+more discussion.
```

Comment:
I guess this is existing text, but I'm not sure RFC8446 actually covers. RFC8446 applies to what TLS 1.3 calls an external PSK. But we've got our own root secret and uses of KDFs. So I think we have to state that rule anew.

(RFC8446 would apply to our derived PSKs, but those satisfy the rules by construction.)

Reply:
Hmm, can you propose text? I'm not sure what specifically which rule you're referring to. (Maybe that each PSK has an associated hash function?)

Reply:
Yeah, that each PSK has an associated hash function applies to IPSKs, not EPSKs. Perhaps we can just replace:

> , as per the rules in Section 4.2.11 from {{!RFC8446}}.

with:

> . This is analogous to the rules in Section 4.2.11 of {{!RFC8446}}.

Reply:
Yeah, that works!